hail/src/main/scala/is/hail/variant/ReferenceGenome.scala

```diff
@@ -434,6 +427,24 @@ case class ReferenceGenome(name: String, contigs: Array[String], lengths: Map[St
     lo.queryInterval(interval, minMatch)
   }
 
+  def heal(tmpdir: String, fs: FS): Unit = this.synchronized {
+    // add liftovers
+    liftoverMaps = Map.empty
+    for ((destRGName, chainFile) <- chainFiles) {
+      val chainFilePath = fs.fileStatus(chainFile).getPath
+      val lo = LiftOver(tmpdir, fs, chainFilePath)
+      liftoverMaps += destRGName -> lo
+    }
+
+    // add sequence
+    fastaReaderCfg = null
```

Comment:
This should also check if the fasta config exists already. I think if we do:
```
addSequence(...)
addLiftover(ctx, file2, rg2)
addLiftover(ctx, file3, rg3)
addLiftover(ctx, file4, rg4)
```

The fasta is re-added for each heal in the liftovers!

Reply:
Given that the fastaReaderCfg is just a case class and everything else going on, it's probably fine to do this, but it's also easy enough to check that we would be creating an equivalent object. done